### PR TITLE
add to scoreboard button, name input and send score to scoreboard

### PR DESCRIPTION
**Pull request by alreuben**
name text input on end game modal

send to scoreboard button on end game modal

JS stringify and send JSON to api

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -175,6 +175,12 @@ li {
     font-size:1rem;
 }
 
+#nameInput {
+    width: 200px;
+    height: 55px
+
```
Comment (HadertS) [inline styles.css]: delete whitespace

File: styles.css [hunk 2]
```diff
@@ -217,4 +223,11 @@ li {
     li {
         font-size: 0.75rem;
     }
+
+    #nameInput {
+        width: 150px;
+        height: 50px
+
```
Comment (HadertS) [inline styles.css]: Delete whitespace

**Review by HadertS — CHANGES_REQUESTED** [review]
Looks good, just a couple of linebreaks that need to be removed in the css

**Review by nesclark — COMMENTED** [review]
looks good

File: index.js [hunk 3]
```diff
@@ -140,3 +140,21 @@ restartButton.addEventListener("click", () => {
     location.reload()
 })
 
+let scoreboardButton = document.querySelector('#scoreboardButton')
+scoreboardButton.addEventListener('click', () => {
+    location.reload()
+    let nameInput = document.querySelector('#nameInput')
+    let dataToSend = {name:nameInput.value,score:scoreCount.value}
+
+    fetch('https://23jancountyscoreboard.dev.io-academy.uk/scores', {
```
Comment (Steph-ski) [inline index.js:147]: should this block be moved over to the left?

**Review by HadertS — APPROVED** [review]

**Pull request merged**